app/scripts/modules/google/loadBalancer/details/deleteModal/deleteModal.controller.ts

```diff
@@ -76,8 +74,6 @@ class DeleteLoadBalancerModalController implements ng.IComponentController {
     } else {
       return () => {
         return this.loadBalancerWriter.deleteLoadBalancer(this.loadBalancer, this.application, {
-          loadBalancerName: this.loadBalancer.name,
-          region: this.loadBalancer.region,
```

Comment:
Deleting a load balancer failed for me. The description validator requires region: https://github.com/spinnaker/clouddriver/blob/master/clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/validators/UpsertGoogleLoadBalancerDescriptionValidator.groovy#L50